pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -210,3 +245,21 @@ func newCDIForCR(cr *hcov1alpha1.HyperConverged) *cdi.CDI {
 		},
 	}
 }
+
+// newNetworkAddonsForCR returns a NetworkAddonsConfig CR
+func newNetworkAddonsForCR(cr *hcov1alpha1.HyperConverged) *networkaddons.NetworkAddonsConfig {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &networkaddons.NetworkAddonsConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   networkaddonsnames.OPERATOR_CONFIG,
+			Labels: labels,
+		},
+		Spec: networkaddons.NetworkAddonsConfigSpec{
+			Multus:          &networkaddons.Multus{},
+			LinuxBridge:     &networkaddons.LinuxBridge{},
```

Comment:
Why not expose Multus and LinuxBridge as configurable options too?

Reply:
HCO should be opinionated. In my opinion Linux Bridge and Multus should be always deployed with this hyperconverged infrastructure.

Reply:
That's fine, but we can default these values behind the scenes.  I'm wondering if we should give the user the opportunity to configure them to be something other than the defaults.

Reply:
Could we start with this and introduce configurable values in a different PR? I think it should be discussed with wider audience from the whole project (and it may also affect kubevirt and CDI). Where is the line between flexibility and "the right way to use hyperconverged cluster".

Reply:
@phoracek sounds good to me